src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -531,8 +531,6 @@ install_packages_in_root (RpmOstreeTreeComposeContext  *self,
       /* Depending on cache state, we may have some pkgs already
        * labeled with a final target policy.
        */
```

Comment:
This comment doesn't apply anymore, right?

Reply:
Yep, deleted thanks!  :arrow_down: